lib/pages/home.dart

```diff
@@ -2,35 +2,69 @@ import 'package:flutter/widgets.dart';
 
 import '../widgets/button.dart';
 import '../widgets/scaffold.dart';
+import '../widgets/bottom_navigate.dart';
 
-class Home extends StatelessWidget {
+import './select_pixel.dart';
+import './feed/pixel_list.dart';
+
+class Home extends StatefulWidget {
+  Home();
   @override
-  Widget build(BuildContext context) {
-    return Scaffold(
-        body: SafeArea(
-      child: Center(
-        child: Column(
-          children: [
-            Text('My Pixels'),
-            Button(
-              label: 'See pixel',
+  _HomeState createState() => _HomeState();
+}
+
+class _HomeState extends State<Home> {
+  static BuildContext _context;
+  static int _currentIndex;
+
+  final List<Widget> _pages = [
+    PixelList(),
+    Text('search'),
+    SelectPixel(),
+    Center(
+      child: Column(
+        children: [
+          Text('My Pixels'),
+          Button(
+            label: 'See pixel',
+            onPressed: () {
+              print('elias');
+              Navigator.of(_context).pushNamed('/pixel_detail');
+            },
+          ),
+          Button(
+              label: 'Send pixel',
               onPressed: () {
-                Navigator.of(context).pushNamed('/pixel_detail');
-              },
-            ),
-            Button(
-                label: 'Send pixel',
-                onPressed: () {
-                  Navigator.of(context).pushNamed('/select_pixel');
-                }),
-            Button(
-                label: 'List pixels',
-                onPressed: () {
-                  Navigator.of(context).pushNamed('/feed');
-                }),
-          ],
-        ),
+                Navigator.of(_context).pushNamed('/select_pixel');
+              }),
+          Button(
+              label: 'List pixels',
+              onPressed: () {
+                Navigator.of(_context).pushNamed('/feed');
+              }),
+        ],
       ),
-    ));
+    ),
+    Text('Profile')
+  ];
+
+  @override
+  void initState() {
+    super.initState();
+    _currentIndex = 0;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    setState(() => {_context = context});
```

Comment:
Hm, I'm not sure if this is a good idea. @erickzanardo what do you think, flutter master?

Reply:
After remove some codes, it's not necessary anymore.